runtime/src/main/cpp/Memory.h

```diff
@@ -548,6 +550,13 @@ void MutationCheck(ObjHeader* obj);
 void FreezeSubgraph(ObjHeader* obj);
 // Ensure this object shall block freezing.
 void EnsureNeverFrozen(ObjHeader* obj);
+// Add TLS object storage, called by the generated code.
+void AddTLSRecord(MemoryState* memory, int module, int size) RUNTIME_NOTHROW;
```

Comment:
Note that using module as a key here is somewhat too restricting.
For example, it is incompatible with fine-grained incremental compilation.
Maybe at least rename `module` to `key` or `id`?

Reply:
Well, this depends on how incremental we are, if codegen phase is being run on module as a whole - it shalln't be a problem.Think it's better describe the current contract in the signature, and fix it when working on IC.